gqlalchemy/vendors/memgraph.py

```diff
@@ -432,3 +450,43 @@ def with_power_bi(self) -> "Memgraph":
         module_name = "power_bi_stream.py"
 
         return self.add_query_module(file_path=file_path, module_name=module_name)
+
+    def get_transactions(self) -> List[MemgraphTransaction]:
+        """Get all transactions in the database.
+        Returns:
+            List[MemgraphTransaction]: A list of MemgraphTransaction objects.
+        """
+
+        transactions_data = self.execute_and_fetch("SHOW TRANSACTIONS;")
+        transactions = []
+
+        for transaction_data in transactions_data:
+            transaction = MemgraphTransaction(
+                username=transaction_data["username"],
+                transaction_id=transaction_data["transaction_id"],
+                query=transaction_data["query"],
+                metadata=transaction_data["metadata"],
+            )
+            transactions.append(transaction)
+
+        return transactions
+
+    def terminate_transactions(self, transaction_ids: List[str]) -> List[MemgraphTerminatedTransaction]:
+        """Terminate transactions in the database.
+        Args:
+            transaction_ids (List[str]): A list of transaction ids to terminate.
+        Returns:
+            List[MemgraphTerminatedTransaction]: A list of MemgraphTerminatedTransaction objects with info on their status.
+        """
+
+        query = (
+            "TERMINATE TRANSACTIONS " + ", ".join([f"'{transaction_id}'" for transaction_id in transaction_ids]) + ";"
+        )
+        print(query)
+        terminated_transactions = []
+        results = self.execute_and_fetch(query)
+
+        for result in results:
+            terminated_transactions.append(MemgraphTerminatedTransaction(result["transaction_id"], result["killed"]))
+        print(terminated_transactions)
```

Comment:
Remove the debug comments:
```suggestion
        terminated_transactions = []
        results = self.execute_and_fetch(query)

        for result in results:
            terminated_transactions.append(MemgraphTerminatedTransaction(result["transaction_id"], result["killed"]))
```